### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bhenhsi**
Adding proposal approval and decision-making process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,18 @@ The design process for changes to M3 is loosely modeled on the [proposal process
 
 - The author (and/or other contributors) do the work as described by the "Implementation" section of the proposal.
 
+## Approval 
+
+The proposal requires approval from all Technical Steering Committee (the “TSC”) members.
```
Comment (mway) [inline README.md]: nit: smart quotes

File: README.md [hunk 2]
```diff
@@ -38,6 +38,18 @@ The design process for changes to M3 is loosely modeled on the [proposal process
...
+
+The current TSC members are:
```
Comment (mway) [inline README.md:45]: can you sort these alphabetically?

also, should we add organizations? e.g. `@mway (Uber)`, `@robskillington (Chronosphere)`?

**Review by mway — COMMENTED** [review]
thank you!

**Review by mway — APPROVED** [review]
lgtm!

**Review by martin-mao — APPROVED** [review]
LGTM

**Pull request merged**